samples/CMakeLists.txt

```diff
@@ -240,7 +240,7 @@ if (NOT ANDROID)
 
     # Sample app specific
     target_link_libraries(frame_generator PRIVATE imageio)
-    target_link_libraries(gltf_viewer PRIVATE gltf-resources gltfio viewer)
+    target_link_libraries(gltf_viewer PRIVATE gltf-resources gltfio viewer imageio)
```

Comment:
Accidentally added this dep, it'll be needed for the next PR though since it will add screenshot capability.